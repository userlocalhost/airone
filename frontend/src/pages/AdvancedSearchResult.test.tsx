/**
 * @jest-environment jsdom
 */

import {
  render,
  waitForElementToBeRemoved,
  screen,
} from "@testing-library/react";
import React from "react";

import { TestWrapper } from "../utils/TestWrapper";

import { AdvancedSearchResults } from "./AdvancedSearchResults";

afterEach(() => {
  jest.clearAllMocks();
});

test("should match snapshot", async () => {
  // TODO avoid to pass the global variable implicitly
  (window as any).django_context = {
    user: {
      is_superuser: false,
    },
  };

  const results = [
    {
      entry: {
        name: "entry1",
      },
      attrs: {
        attr1: {
          type: 1,
          value: "attr1",
        },
        attr2: {
          type: 1,
          value: "attr1",
        },
      },
    },
  ];

  /* eslint-disable */
  jest
    .spyOn(require("../utils/AironeAPIClient"), "searchEntries")
    .mockResolvedValue({
      json() {
        return Promise.resolve({
          result: {
            ret_values: results,
          },
        });
      },
    });
  /* eslint-enable */

  // wait async calls and get rendered fragment
  const result = render(<AdvancedSearchResults />, {
    wrapper: TestWrapper,
  });
  await waitForElementToBeRemoved(screen.getByTestId("loading"));

  expect(result).toMatchSnapshot();
});
