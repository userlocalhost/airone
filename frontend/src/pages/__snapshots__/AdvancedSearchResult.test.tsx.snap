// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should match snapshot 1`] = `
Object {
  "asFragment": [Function],
  "baseElement": <body>
    <div>
      <div
        class="container-fluid MuiBox-root css-0"
      >
        <nav
          aria-label="breadcrumb"
          class="MuiTypography-root MuiTypography-body1 MuiBreadcrumbs-root makeStyles-breadcrumbs-2 css-63vwl1-MuiTypography-root-MuiBreadcrumbs-root"
        >
          <ol
            class="MuiBreadcrumbs-ol css-4pdmu4-MuiBreadcrumbs-ol"
          >
            <li
              class="MuiBreadcrumbs-li"
            >
              <a
                class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                href="/new-ui/"
              >
                Top
              </a>
            </li>
            <li
              aria-hidden="true"
              class="MuiBreadcrumbs-separator css-1wuw8dw-MuiBreadcrumbs-separator"
            >
              /
            </li>
            <li
              class="MuiBreadcrumbs-li"
            >
              <p
                class="MuiTypography-root MuiTypography-body1 css-1gqyz35-MuiTypography-root"
              >
                検索結果
              </p>
            </li>
          </ol>
        </nav>
        <div
          class="MuiBox-root css-1ynyhby"
        >
          <p
            class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
          >
            検索結果: (
            1
             件)
          </p>
          <button
            class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root makeStyles-button-1 css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            <span
              class="MuiButton-startIcon MuiButton-iconSizeMedium css-1d6wzja-MuiButton-startIcon"
            >
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                data-testid="SettingsIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M19.14 12.94c.04-.3.06-.61.06-.94 0-.32-.02-.64-.07-.94l2.03-1.58c.18-.14.23-.41.12-.61l-1.92-3.32c-.12-.22-.37-.29-.59-.22l-2.39.96c-.5-.38-1.03-.7-1.62-.94l-.36-2.54c-.04-.24-.24-.41-.48-.41h-3.84c-.24 0-.43.17-.47.41l-.36 2.54c-.59.24-1.13.57-1.62.94l-2.39-.96c-.22-.08-.47 0-.59.22L2.74 8.87c-.12.21-.08.47.12.61l2.03 1.58c-.05.3-.09.63-.09.94s.02.64.07.94l-2.03 1.58c-.18.14-.23.41-.12.61l1.92 3.32c.12.22.37.29.59.22l2.39-.96c.5.38 1.03.7 1.62.94l.36 2.54c.05.24.24.41.48.41h3.84c.24 0 .44-.17.47-.41l.36-2.54c.59-.24 1.13-.56 1.62-.94l2.39.96c.22.08.47 0 .59-.22l1.92-3.32c.12-.22.07-.47-.12-.61l-2.01-1.58zM12 15.6c-1.98 0-3.6-1.62-3.6-3.6s1.62-3.6 3.6-3.6 3.6 1.62 3.6 3.6-1.62 3.6-3.6 3.6z"
                />
              </svg>
            </span>
            高度な検索
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
          <button
            class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root makeStyles-button-1 css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            YAML 出力
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
          <button
            class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root makeStyles-button-1 css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
            tabindex="0"
            type="button"
          >
            CSV 出力
            <span
              class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
            />
          </button>
        </div>
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1ps6pg7-MuiPaper-root"
        >
          <div
            class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiTableContainer-root css-11xur9t-MuiPaper-root-MuiTableContainer-root"
          >
            <table
              class="MuiTable-root css-rqglhn-MuiTable-root"
            >
              <thead
                class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
              >
                <tr
                  class="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
                >
                  <th
                    class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                    scope="col"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                    >
                      Name
                    </p>
                    <div
                      class="MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary css-a88p61-MuiInputBase-root-MuiInput-root"
                    >
                      <input
                        class="MuiInput-input MuiInputBase-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
                        placeholder="絞り込む"
                        type="text"
                        value=""
                      />
                    </div>
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                    scope="col"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                    >
                      attr1
                    </p>
                    <div
                      class="MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary css-a88p61-MuiInputBase-root-MuiInput-root"
                    >
                      <input
                        class="MuiInput-input MuiInputBase-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
                        placeholder="絞り込む"
                        type="text"
                        value=""
                      />
                    </div>
                  </th>
                  <th
                    class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                    scope="col"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                    >
                      attr2
                    </p>
                    <div
                      class="MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary css-a88p61-MuiInputBase-root-MuiInput-root"
                    >
                      <input
                        class="MuiInput-input MuiInputBase-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
                        placeholder="絞り込む"
                        type="text"
                        value=""
                      />
                    </div>
                  </th>
                </tr>
              </thead>
              <tbody
                class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
              >
                <tr
                  class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
                >
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                  >
                    <p
                      class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                    >
                      entry1
                    </p>
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                  >
                    <a
                      href="/new-ui/entries/undefined/show"
                    />
                  </td>
                  <td
                    class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                  >
                    <a
                      href="/new-ui/entries/undefined/show"
                    />
                  </td>
                </tr>
              </tbody>
            </table>
          </div>
          <div
            class="MuiTablePagination-root css-jtlhu6-MuiTablePagination-root"
          >
            <div
              class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular MuiTablePagination-toolbar css-11bfvty-MuiToolbar-root-MuiTablePagination-toolbar"
            >
              <div
                class="MuiTablePagination-spacer css-1psng7p-MuiTablePagination-spacer"
              />
              <p
                class="MuiTablePagination-selectLabel css-pdct74-MuiTablePagination-selectLabel"
                id="mui-2"
              >
                Rows per page:
              </p>
              <div
                class="MuiInputBase-root MuiInputBase-colorPrimary css-16c50h-MuiInputBase-root-MuiTablePagination-select"
              >
                <div
                  aria-expanded="false"
                  aria-haspopup="listbox"
                  aria-labelledby="mui-2 mui-1"
                  class="MuiTablePagination-select MuiSelect-select MuiSelect-standard MuiInputBase-input css-194a1fa-MuiSelect-select-MuiInputBase-input"
                  id="mui-1"
                  role="button"
                  tabindex="0"
                >
                  100
                </div>
                <input
                  aria-hidden="true"
                  class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
                  tabindex="-1"
                  value="100"
                />
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTablePagination-selectIcon MuiSelect-icon MuiSelect-iconStandard css-pqjvzy-MuiSvgIcon-root-MuiSelect-icon"
                  data-testid="ArrowDropDownIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M7 10l5 5 5-5z"
                  />
                </svg>
              </div>
              <p
                class="MuiTablePagination-displayedRows css-levciy-MuiTablePagination-displayedRows"
              >
                1–1 of 1
              </p>
              <div
                class="MuiTablePagination-actions"
              >
                <button
                  aria-label="Go to previous page"
                  class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-colorInherit MuiIconButton-sizeMedium css-zylse7-MuiButtonBase-root-MuiIconButton-root"
                  disabled=""
                  tabindex="-1"
                  title="Go to previous page"
                  type="button"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="KeyboardArrowLeftIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M15.41 16.09l-4.58-4.59 4.58-4.59L14 5.5l-6 6 6 6z"
                    />
                  </svg>
                </button>
                <button
                  aria-label="Go to next page"
                  class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-colorInherit MuiIconButton-sizeMedium css-zylse7-MuiButtonBase-root-MuiIconButton-root"
                  disabled=""
                  tabindex="-1"
                  title="Go to next page"
                  type="button"
                >
                  <svg
                    aria-hidden="true"
                    class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                    data-testid="KeyboardArrowRightIcon"
                    focusable="false"
                    viewBox="0 0 24 24"
                  >
                    <path
                      d="M8.59 16.34l4.58-4.59-4.58-4.59L10 5.75l6 6-6 6z"
                    />
                  </svg>
                </button>
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
  </body>,
  "container": <div>
    <div
      class="container-fluid MuiBox-root css-0"
    >
      <nav
        aria-label="breadcrumb"
        class="MuiTypography-root MuiTypography-body1 MuiBreadcrumbs-root makeStyles-breadcrumbs-2 css-63vwl1-MuiTypography-root-MuiBreadcrumbs-root"
      >
        <ol
          class="MuiBreadcrumbs-ol css-4pdmu4-MuiBreadcrumbs-ol"
        >
          <li
            class="MuiBreadcrumbs-li"
          >
            <a
              class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
              href="/new-ui/"
            >
              Top
            </a>
          </li>
          <li
            aria-hidden="true"
            class="MuiBreadcrumbs-separator css-1wuw8dw-MuiBreadcrumbs-separator"
          >
            /
          </li>
          <li
            class="MuiBreadcrumbs-li"
          >
            <p
              class="MuiTypography-root MuiTypography-body1 css-1gqyz35-MuiTypography-root"
            >
              検索結果
            </p>
          </li>
        </ol>
      </nav>
      <div
        class="MuiBox-root css-1ynyhby"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
        >
          検索結果: (
          1
           件)
        </p>
        <button
          class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root makeStyles-button-1 css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          <span
            class="MuiButton-startIcon MuiButton-iconSizeMedium css-1d6wzja-MuiButton-startIcon"
          >
            <svg
              aria-hidden="true"
              class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
              data-testid="SettingsIcon"
              focusable="false"
              viewBox="0 0 24 24"
            >
              <path
                d="M19.14 12.94c.04-.3.06-.61.06-.94 0-.32-.02-.64-.07-.94l2.03-1.58c.18-.14.23-.41.12-.61l-1.92-3.32c-.12-.22-.37-.29-.59-.22l-2.39.96c-.5-.38-1.03-.7-1.62-.94l-.36-2.54c-.04-.24-.24-.41-.48-.41h-3.84c-.24 0-.43.17-.47.41l-.36 2.54c-.59.24-1.13.57-1.62.94l-2.39-.96c-.22-.08-.47 0-.59.22L2.74 8.87c-.12.21-.08.47.12.61l2.03 1.58c-.05.3-.09.63-.09.94s.02.64.07.94l-2.03 1.58c-.18.14-.23.41-.12.61l1.92 3.32c.12.22.37.29.59.22l2.39-.96c.5.38 1.03.7 1.62.94l.36 2.54c.05.24.24.41.48.41h3.84c.24 0 .44-.17.47-.41l.36-2.54c.59-.24 1.13-.56 1.62-.94l2.39.96c.22.08.47 0 .59-.22l1.92-3.32c.12-.22.07-.47-.12-.61l-2.01-1.58zM12 15.6c-1.98 0-3.6-1.62-3.6-3.6s1.62-3.6 3.6-3.6 3.6 1.62 3.6 3.6-1.62 3.6-3.6 3.6z"
              />
            </svg>
          </span>
          高度な検索
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <button
          class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root makeStyles-button-1 css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          YAML 出力
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
        <button
          class="MuiButton-root MuiButton-outlined MuiButton-outlinedPrimary MuiButton-sizeMedium MuiButton-outlinedSizeMedium MuiButtonBase-root makeStyles-button-1 css-1rwt2y5-MuiButtonBase-root-MuiButton-root"
          tabindex="0"
          type="button"
        >
          CSV 出力
          <span
            class="MuiTouchRipple-root css-8je8zh-MuiTouchRipple-root"
          />
        </button>
      </div>
      <div
        class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 css-1ps6pg7-MuiPaper-root"
      >
        <div
          class="MuiPaper-root MuiPaper-elevation MuiPaper-rounded MuiPaper-elevation1 MuiTableContainer-root css-11xur9t-MuiPaper-root-MuiTableContainer-root"
        >
          <table
            class="MuiTable-root css-rqglhn-MuiTable-root"
          >
            <thead
              class="MuiTableHead-root css-15wwp11-MuiTableHead-root"
            >
              <tr
                class="MuiTableRow-root MuiTableRow-head css-1q1u3t4-MuiTableRow-root"
              >
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                  scope="col"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                  >
                    Name
                  </p>
                  <div
                    class="MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary css-a88p61-MuiInputBase-root-MuiInput-root"
                  >
                    <input
                      class="MuiInput-input MuiInputBase-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
                      placeholder="絞り込む"
                      type="text"
                      value=""
                    />
                  </div>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                  scope="col"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                  >
                    attr1
                  </p>
                  <div
                    class="MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary css-a88p61-MuiInputBase-root-MuiInput-root"
                  >
                    <input
                      class="MuiInput-input MuiInputBase-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
                      placeholder="絞り込む"
                      type="text"
                      value=""
                    />
                  </div>
                </th>
                <th
                  class="MuiTableCell-root MuiTableCell-head MuiTableCell-sizeMedium css-1ygcj2i-MuiTableCell-root"
                  scope="col"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                  >
                    attr2
                  </p>
                  <div
                    class="MuiInput-root MuiInput-underline MuiInputBase-root MuiInputBase-colorPrimary css-a88p61-MuiInputBase-root-MuiInput-root"
                  >
                    <input
                      class="MuiInput-input MuiInputBase-input css-1x51dt5-MuiInputBase-input-MuiInput-input"
                      placeholder="絞り込む"
                      type="text"
                      value=""
                    />
                  </div>
                </th>
              </tr>
            </thead>
            <tbody
              class="MuiTableBody-root css-apqrd9-MuiTableBody-root"
            >
              <tr
                class="MuiTableRow-root css-1q1u3t4-MuiTableRow-root"
              >
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                >
                  <p
                    class="MuiTypography-root MuiTypography-body1 css-ahj2mt-MuiTypography-root"
                  >
                    entry1
                  </p>
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                >
                  <a
                    href="/new-ui/entries/undefined/show"
                  />
                </td>
                <td
                  class="MuiTableCell-root MuiTableCell-body MuiTableCell-sizeMedium css-1ex1afd-MuiTableCell-root"
                >
                  <a
                    href="/new-ui/entries/undefined/show"
                  />
                </td>
              </tr>
            </tbody>
          </table>
        </div>
        <div
          class="MuiTablePagination-root css-jtlhu6-MuiTablePagination-root"
        >
          <div
            class="MuiToolbar-root MuiToolbar-gutters MuiToolbar-regular MuiTablePagination-toolbar css-11bfvty-MuiToolbar-root-MuiTablePagination-toolbar"
          >
            <div
              class="MuiTablePagination-spacer css-1psng7p-MuiTablePagination-spacer"
            />
            <p
              class="MuiTablePagination-selectLabel css-pdct74-MuiTablePagination-selectLabel"
              id="mui-2"
            >
              Rows per page:
            </p>
            <div
              class="MuiInputBase-root MuiInputBase-colorPrimary css-16c50h-MuiInputBase-root-MuiTablePagination-select"
            >
              <div
                aria-expanded="false"
                aria-haspopup="listbox"
                aria-labelledby="mui-2 mui-1"
                class="MuiTablePagination-select MuiSelect-select MuiSelect-standard MuiInputBase-input css-194a1fa-MuiSelect-select-MuiInputBase-input"
                id="mui-1"
                role="button"
                tabindex="0"
              >
                100
              </div>
              <input
                aria-hidden="true"
                class="MuiSelect-nativeInput css-yf8vq0-MuiSelect-nativeInput"
                tabindex="-1"
                value="100"
              />
              <svg
                aria-hidden="true"
                class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium MuiTablePagination-selectIcon MuiSelect-icon MuiSelect-iconStandard css-pqjvzy-MuiSvgIcon-root-MuiSelect-icon"
                data-testid="ArrowDropDownIcon"
                focusable="false"
                viewBox="0 0 24 24"
              >
                <path
                  d="M7 10l5 5 5-5z"
                />
              </svg>
            </div>
            <p
              class="MuiTablePagination-displayedRows css-levciy-MuiTablePagination-displayedRows"
            >
              1–1 of 1
            </p>
            <div
              class="MuiTablePagination-actions"
            >
              <button
                aria-label="Go to previous page"
                class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-colorInherit MuiIconButton-sizeMedium css-zylse7-MuiButtonBase-root-MuiIconButton-root"
                disabled=""
                tabindex="-1"
                title="Go to previous page"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="KeyboardArrowLeftIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M15.41 16.09l-4.58-4.59 4.58-4.59L14 5.5l-6 6 6 6z"
                  />
                </svg>
              </button>
              <button
                aria-label="Go to next page"
                class="MuiButtonBase-root Mui-disabled MuiIconButton-root Mui-disabled MuiIconButton-colorInherit MuiIconButton-sizeMedium css-zylse7-MuiButtonBase-root-MuiIconButton-root"
                disabled=""
                tabindex="-1"
                title="Go to next page"
                type="button"
              >
                <svg
                  aria-hidden="true"
                  class="MuiSvgIcon-root MuiSvgIcon-fontSizeMedium css-i4bv87-MuiSvgIcon-root"
                  data-testid="KeyboardArrowRightIcon"
                  focusable="false"
                  viewBox="0 0 24 24"
                >
                  <path
                    d="M8.59 16.34l4.58-4.59-4.58-4.59L10 5.75l6 6-6 6z"
                  />
                </svg>
              </button>
            </div>
          </div>
        </div>
      </div>
    </div>
  </div>,
  "debug": [Function],
  "findAllByAltText": [Function],
  "findAllByDisplayValue": [Function],
  "findAllByLabelText": [Function],
  "findAllByPlaceholderText": [Function],
  "findAllByRole": [Function],
  "findAllByTestId": [Function],
  "findAllByText": [Function],
  "findAllByTitle": [Function],
  "findByAltText": [Function],
  "findByDisplayValue": [Function],
  "findByLabelText": [Function],
  "findByPlaceholderText": [Function],
  "findByRole": [Function],
  "findByTestId": [Function],
  "findByText": [Function],
  "findByTitle": [Function],
  "getAllByAltText": [Function],
  "getAllByDisplayValue": [Function],
  "getAllByLabelText": [Function],
  "getAllByPlaceholderText": [Function],
  "getAllByRole": [Function],
  "getAllByTestId": [Function],
  "getAllByText": [Function],
  "getAllByTitle": [Function],
  "getByAltText": [Function],
  "getByDisplayValue": [Function],
  "getByLabelText": [Function],
  "getByPlaceholderText": [Function],
  "getByRole": [Function],
  "getByTestId": [Function],
  "getByText": [Function],
  "getByTitle": [Function],
  "queryAllByAltText": [Function],
  "queryAllByDisplayValue": [Function],
  "queryAllByLabelText": [Function],
  "queryAllByPlaceholderText": [Function],
  "queryAllByRole": [Function],
  "queryAllByTestId": [Function],
  "queryAllByText": [Function],
  "queryAllByTitle": [Function],
  "queryByAltText": [Function],
  "queryByDisplayValue": [Function],
  "queryByLabelText": [Function],
  "queryByPlaceholderText": [Function],
  "queryByRole": [Function],
  "queryByTestId": [Function],
  "queryByText": [Function],
  "queryByTitle": [Function],
  "rerender": [Function],
  "unmount": [Function],
}
`;
